### ISSUE TITLE
Plugin only uses link to first recording element

### ISSUE THREAD

**Issue by Be-El** [blindsidenetworks/mattermost-plugin-bigbluebutton#88]
We are using a little cluster of BBB backend and a scalelite load balancer. We have configured to backend to record all possible sources (presentation, audio, video, screenshare, notes) by installing the corresponding plugins.

The plugin works in this setup, but only uses the first available format. In most cases this is not the expected one (see https://github.com/blindsidenetworks/mattermost-plugin-bigbluebutton/blob/master/server/responsehandlers.go#L544).

It seems that the API between the server side and the web client only supports a single link; in case of our setup we need a list of links for the different types. The thumbnails are also broken in most cases, since they also only refer to the first link without taking its format into account.

**Comment by harshilsharma63** [comment]
@Be-El the recording link your receive in MM shows both screens shared webcam recording. Is that not the case with your setup?

**Comment by greatfireball** [comment]
Indeed, in our setup the Podcast is always the first recording returened and therefore will be linked here. I created a fix which displays different types. I can create a pull request if you would like to.

**Comment by Be-El** [comment]
Our setup is also not a simple BBB, but a scalelite load balancer and several BBB endpoints. The API requests to retrieve the recordings are sent to scalelite; maybe the API or the scalelite collector uses a different order in the underlying XML file.

**Comment by harshilsharma63** [comment]
@greatfireball feel free to create a pull request.

@Be-El the plugin doesn't fetch the recordings, the BBB server pushes the recordings to the plugin.
